nodejs.org/test/run

```diff
@@ -101,6 +101,11 @@ test_s2i_usage() {
   s2i usage ${s2i_args} ${IMAGE_TAG} &>/dev/null
 }
 
+test_s2i_incremental() {
+  echo "Testing s2i build incremental..."
+  s2i build --incremental=true ${s2i_args} file://${test_dir}/test-app ${IMAGE_TAG} ${IMAGE_NAME}-testapp
```

Comment:
I agree.
But look the other comment, I did a usage test and maybe this is not working :/